### ISSUE TITLE
Chapter 13 typo

### ISSUE THREAD

**Issue by rickiepark** [rasbt/python-machine-learning-book-2nd-edition#100]
(p441) In 3rd paragraph, "we can set values for the **weight** decay contant.." should be "we can set values for the **learning rate** decay contant..".
As you know, SGD's decay param is learning rate decay, not weight decay.

**Comment by vmirly** [comment]
Thank you for your comment! Yes, that's right! In Keras the decay parameter is for the learning rate decay, whereas in other frameworks (like pytorch for example) they also have weight_decay parameter! We will fix that in future editions. 

Thanks again!

**Comment by rickiepark** [comment]
(p446) In softmax eq., \sum_{i=1}^M e^{z_j} should be \sum_{j=1}^M e^{z_j}

Thanks.
